src/main/java/net/snowflake/client/jdbc/SnowflakeUtil.java

```diff
@@ -254,9 +254,9 @@ static public SnowflakeColumnMetadata extractColumnMetadata(
         break;
 
       default:
-        throw new SnowflakeSQLException(SqlState.INTERNAL_ERROR,
+        throw new SnowflakeSQLLoggedException(SqlState.INTERNAL_ERROR,
```

Comment:
session can be passed in https://github.com/snowflakedb/snowflake-jdbc/blob/869cde8ed1f6ae1a311a9ff4506a89657c972bea/src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java#L1933

Reply:
added session

Reply:
But what about others in this file?